App/components/SearchInput/input.story.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { action } from '@storybook/addon-actions'
+import { boolean, select, text, number } from '@storybook/addon-knobs'
+import { storiesOf } from '@storybook/react-native'
+import { BufferView, ThemeDecorator } from '../../config/decorators'
+import { SearchInput } from './input'
+import { INPUT_SIZE_KEYS } from '../../theme/inputSearchSizes'
+
+const StyledWrapper = styled.View`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledPadding = styled.View`
+  margin-top: 5px;
+  margin-bottom: 5px;
+`
+
+export const list = () => (
+  <StyledWrapper>
+    <SearchInput
+      onChange={() => 's'}
```

Comment:
here and after that, dont catch what does that means, but probably like a 'noop' function. Maybe onChange is optional?

Reply:
All questions to @Maximus135